sos/plugins/iprconfig.py

```diff
@@ -10,20 +10,18 @@
 
 import re
 from sos.plugins import Plugin, RedHatPlugin, UbuntuPlugin, DebianPlugin
-from sos.utilities import is_executable
 
 
 class IprConfig(Plugin, RedHatPlugin, UbuntuPlugin, DebianPlugin):
     """IBM Power RAID storage adapter configuration information
     """
 
     plugin_name = 'iprconfig'
-
-    def check_enabled(self):
-        arch = self.policy.get_arch()
-        return "ppc64" in arch and is_executable("iprconfig")
```

Comment:
I'm not sure we're really doing the right thing here - the restriction to PPC came with the original patch from IBM back in 2014, and the "IPR" branding originally meant "IBM Power RAID". Browsing some hardware articles on IBM's site this still seems to be the case, but I'm confused since it appears all the major distros (us, inc. Fedora/CentOS etc., Debian, SuSE, Ubuntu) all build the package for x86_64 as well.

Interestingly I notice that the packages currently built for Fedora include an `iprsos` command that "is used to generate an sosreport-like report by collecting hardware information from the system, along with other system calls such as lspci, lscsi, and others."

If nothing else I would say that's strong evidence that we need the `lspci` data on Power systems at a minimum...

And sure enough my x86_64 kernel has a module built for "IBM Power RAID" devices...

```$ modinfo ipr
filename:       /lib/modules/5.4.7-100.fc30.x86_64/kernel/drivers/scsi/ipr.ko.xz
version:        2.6.4
license:        GPL
description:    IBM Power RAID SCSI Adapter Driver
author:         Brian King <brking@us.ibm.com>
srcversion:     BAB381AAD15DAC158FA9B8B
...
```
I think ultimately we may need to ask IBM to clarify the support status here, and to help guide us on these decisions. It's really unclear from the publicly available documentation, and it's almost impossible for us to get access to the complete range of possible configurations to find out for ourselves.